src/sentry/api/event_search.py

```diff
@@ -1633,31 +1643,68 @@ def parse_function(field, match=None):
     )
 
 
+def count_required_arguments(function):
+    n = 0
+    for arg in function["args"]:
+        if arg.has_default:
+            break
+        n += 1
+    return n
+
+
+def count_total_arguments(function):
+    return len(function["args"])
```

Comment:
Why do you need a function for this?

Reply:
No reason beyond it looks nicer syntactically